flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveSourceBuilder.java

```diff
@@ -315,7 +315,7 @@ private RowType getProducedRowType() {
         return (RowType) producedSchema.toRowDataType().bridgedTo(RowData.class).getLogicalType();
     }
 
-    private BulkFormat<RowData, HiveSourceSplit> createDefaultBulkFormat() {
+    public BulkFormat<RowData, HiveSourceSplit> createDefaultBulkFormat() {
```

Comment:
ditto

Reply:
> ditto

I think this modify is necessary. Because $createDefaultBulkFormat() can only used by $buildWithDefaultBulkFormat(), however $buildWithDefaultBulkFormat() a HiveSource not a BulkFormat. So, it should be a public method  which can be used to create default bulk format.